sdk/core/core-paging/package.json

```diff
@@ -14,9 +10,11 @@
   ],
   "keywords": [
     "microsoft",
-    "clientruntime"
+    "clientruntime",
+    "azure",
+    "cloud"
   ],
-  "main": "./dist-esm/index.js",
+  "main": "dist/index.js",
```

Comment:
Nothing in this package generates a `dist` folder, so not sure where this would be coming from?